lte/cloud/configs/subscriberdb.yml

```diff
@@ -12,3 +12,7 @@
 
 # flatDigestEnabled is a feature flag for the flat digest functionality.
 flatDigestEnabled: true
+
+# syncInterval is the interval in seconds between gateway requests to sync its
+# subscriberdb with the cloud.
+syncInterval: 300
```

Comment:
This is actually the "default" sync interval, so make that clear in the var name and docstring, as well as where the value gets pulled in (service config)